ethereum/deployments.ts

```diff
@@ -55,10 +55,10 @@ export const deployments: RigsDeployments = {
     tablelandHost: "https://testnet.tableland.network",
     contractTable: "rigs_contract_80001_3819",
     allowlistTable: "rigs_allowlist_80001_3820",
-    partsTable: "parts_80001_3859",
-    layersTable: "layers_80001_3860",
-    attributesTable: "rig_attributes_80001_3861",
-    lookupsTable: "lookups_80001_4249",
+    partsTable: "parts_80001_4038",
+    layersTable: "layers_80001_4039",
+    attributesTable: "rig_attributes_80001_4040",
+    lookupsTable: "lookups_80001_4041",
```

Comment:
These are the "official" tables. Importantly, the lookups table contains the same values for the animation base url as mainnet, so for now the marketplaces with mumbai testnet support will display the wrong rig (the mainnet one) for the animation url. That is ok for now because i just wanted to have proper tables in here that are queryable. We can discuss the contents of the lookups table as a separate issue, come up with a strategy, and get that situation fixed asap.

Reply:
Issue to track that https://github.com/tablelandnetwork/rigs/issues/577